src/Generation/Descriptions/treasure_play/context/query_text

```diff
@@ -0,0 +1,4 @@
+The player has stepped into a place, which contains a treasure chest.
+Based on the following intuitons generate a context message that can be prompted to the player. 
```

Comment:
It might be a good idea to rename intuitions to something else, or to explain in prompt what it means. The LLM only knows the dictionary definition of the word, which is not very helpful here.

Reply:
Haha, I have no idea why I thought "intuition" was synonymuos with "circumstance." It seems my linguistic knowledge has failed me.

I'll probably rewrite it everywhere just to be more consistent